src/api/nodeRpcCall.js

```diff
@@ -1877,6 +1877,32 @@ const congressProposeLegislationReferendum = async (
   return await submitExtrinsic( extrinsic, walletAddress);
 }
 
+const congressSendTreasuryLld = async (transferToAddress, transferAmount, walletAddress, callback) => {
+  const api = await getApi();
+  const injector = await web3FromAddress(walletAddress);
+
+  const threshold = await congressMajorityThreshold();
+  const proposal = api.tx.treasury.spend(transferAmount, transferToAddress);
+  const length = proposal.length;
+  
+  const extrinsic = api.tx.council.propose(threshold, proposal, length);
+  extrinsic.signAndSend(walletAddress, { signer: injector.signer }, ({ status, events, dispatchError }) => {
```

Comment:
As we'll need updates to this PR either way, can you update this to new convention? So:
* `submitExtrinsic` here in nodeRpcCall
* worker in sagas that uses `call` instead of `cps` and doesn't do explicit error checking
* `blockchainWatcher` for watcher in sagas.